### DISCUSSION TITLE
Spatie Medial Library pro responsive  images are not refreshing with Livewire

### DISCUSSION THREAD

**Discussion by faxunil** [livewire/livewire#4687] [1. Help] [answered]
Hi!
I have a blog, where I'm using Spatie Media Library Pro whit 
``` {{ $banner->getFirstMedia('banner')
            ->img('default', ['alt'=>$banner->hint,'loading'=>'lazy']) }} ``
for displaying the images.

If I'm using pager by Livewire on the list of post, the new pages loaded are not displaying the pictrures only the worst img from the images set with blur. You can't recognise the pictrue. If you refereshe browser the images are loading.

Same issue If  have a carousel where the sliders are loaded with Livewie wire:poll  mehode  only blured image  is loading.

Do YOu have any ideas how to solve this issue?

Thanks

Gergely

https://spatie.be/docs/laravel-medialibrary/v9/responsive-images/getting-started-with-responsive-images

**Comment by joshhanley** [comment]
@faxunil please fix the formatting of your post as it's hard to read.Thanks!

**Reply by faxunil** [reply]
Sorry. And thank for Your help.

**Comment by silnarm** [comment, marked as answer]
There is a discussion on the base (non-pro version) github page for this lib about this problem:
https://github.com/spatie/laravel-medialibrary/discussions/2548

That is also probably/possibly a better place to continue the discussion and/or ask other questions.

**Reply by faxunil** [reply]
YEEEEES! This solved my problem, thanks a lot.
I find it very strange and ugly, that Spatie guys not solved this issue since a year......
